virtcontainers/kata_agent.go

```diff
@@ -73,16 +74,17 @@ func (s *kataVSOCK) String() string {
 // KataAgentState is the structure describing the data stored from this
 // agent implementation.
 type KataAgentState struct {
-	ProxyPid     int
-	ProxyBuiltIn bool
```

Comment:
I see that  you moved the ProxyBuiltIn from KataAgentState to kataAgent. Can you provide an explanation in the commit message, maybe move this to a separate commit.